_includes/i18n_dropdown_mobile.html

```diff
@@ -2,7 +2,7 @@
   <div class="sm-hide border-bottom border-blue">
     <div class="container cntnr-wide py1 px2 lg-px0 h5">
       <div class="center">
-        <span id="i18n-mobile-toggle" class="block text-decoration-none white fs-13p">
+        <span id="i18n-mobile-toggle" class="block text-decoration-none white fs-13p" role="button" tabindex="0">
```

Comment:
`tabindex="0"` is just a trick to get elements that aren't links (like divs and spans) to be focusable when tabbing through the page, since only one of the menus are visible at a time having this on both shouldn't have any negative effect. Am I following your comment correctly?